CONTRIBUTING.md

```diff
@@ -95,9 +98,20 @@ To run the tests using a `pipenv` environment:
   - Run `make test-with-html-report` to generate test coverage report in `htmlcov/inxdex.html`
 
 # Using the Databricks Labs data generator
-To use the project, the generated wheel should be installed in your Python notebook as a wheel based library
+The release binaries can be accessed at:
+- Databricks Labs Github Data Generator releases - https://github.com/databrickslabs/dbldatagen/releases
+
+To use download a wheel file and install using the Databricks install mechanism to install a wheel based
+library into your workspace.
+
+Alternatively, you can install the library as a notebook scoped library when working within the Databricks 
+notebook environment through the use of a `%pip` cell in your notebook.
+
+To install as a notebook-scoped library, create and execute a notebook cell with the following text:
+
+> `%pip install git+https://github.com/databrickslabs/dbldatagen`
```

Comment:
it also easier to point to the binary 

```
%pip install https://github.com/databrickslabs/dbldatagen/releases/download/v021/dbldatagen-0.2.1-py3-none-any.whl
```

Reply:
The correct way to install binary packages is for us to deploy releases to PyPi - but I am still waiting to get sign off on that